test/e2e/fixture/fixture.go

```diff
@@ -34,13 +34,14 @@ import (
 )
 
 const (
-	defaultAriServer = "localhost:8080"
+	defaultApiServer = "localhost:8080"
 	adminPassword    = "password"
 	testingLabel     = "e2e.argoproj.io"
 	ArgoCDNamespace  = "argocd-e2e"
 
 	// ensure all repos are in one directory tree, so we can easily clean them up
-	tmpDir = "/tmp/argo-e2e"
+	tmpDir  = "/tmp/argo-e2e"
+	repoDir = "testdata.git"
```

Comment:
weird name for a directory

Reply:
This name eventually became part of https git url: https://localhost:9443/argo-e2e/testdata.git

I had to add `.git` postfix because both argocd and kustomize normalize https git url and add `.git`.